city_scrapers/spiders/chi_ssa_52.py

```diff
@@ -0,0 +1,168 @@
+
+                meeting["status"] = self._get_status(meeting)
+                meeting["id"] = self._get_id(meeting)
+
+                yield meeting
+
+    def _replace_meet_chars(self, meet):
+        meet = meet.css("::text").get()
+        meet = meet.replace(".", "")
+        meet = meet.replace("\xa0", "")
+        meet = meet.replace("…", "")
+        meet = meet.replace(",", "")
+        meet = meet.strip()
+
+        return meet
+
+    def _clean_meet(self, meet):
+        """Clean and parse a possible meet <li> item"""
+        payload = {}
+        meet = self._replace_meet_chars(meet)
+        meet = meet.split()
+
+        if len(meet) == 4:
+            day_week = meet[0]
+            month = meet[1]
+            month_day = meet[2]
+            year = meet[3][:4]
+            time = meet[3][4:]
+            payload = {
+                "day_week": day_week,
+                "month": month,
+                "month_day": month_day,
+                "year": year,
+                "time": time,
+            }
+
+        if len(meet) == 6:
+            day_week = meet[0]
+            month = meet[1]
+            month_day = meet[2]
+            year = meet[3][:4]
+            time = meet[3][4:]
+            info = " ".join(meet[4:])
+            payload = {
+                "day_week": day_week,
+                "month": month,
+                "month_day": month_day,
+                "year": year,
+                "time": time,
+                "info": info,
+            }
+
+        return payload
+
+    def _parse_title(self, item):
+        """Parse or generate meeting title."""
+        item = item.replace("\xa0", "").strip()
+        return item
+
+    def _parse_description(self, item):
+        """Parse or generate meeting description."""
+        return ""
+
+    def _parse_classification(self, item):
+        """Parse or generate classification from allowed options."""
+        return NOT_CLASSIFIED
+
+    def _parse_start(self, item):
+        """Parse start datetime as a naive datetime object."""
+        months = [
+            "JANUARY",
+            "FEBRUARY",
+            "MARCH",
+            "APRIL",
+            "MAY",
+            "JUNE",
+            "JULY",
+            "AUGUST",
+            "SEPTEMBER",
+            "OCTOBER",
+            "NOVEMBER",
+            "DECEMBER",
+        ]
+
+        try:
+            date = datetime.strptime(
+                f"{item['month_day']} {item['month']}, {item['year']}", "%d %B, %Y"
+            )
+        except ValueError:
+            for month in months:
+                ratio = SequenceMatcher(None, month, item["month"]).ratio()
+                if ratio > 0.5:
+                    date = datetime.strptime(
+                        f"{item['month_day']} {month}, {item['year']}", "%d %B, %Y"
+                    )
+
+        time = item["time"].split(":")  # [0] = hours, [1] = minutes
+        date = date.replace(hour=int(time[0]), minute=int(time[1]))
```

Comment:
It looks like 1:30 is in the afternoon here, so we wouldn't want to use the integers directly. We can use `datetime.strptime` similar to the way you are above, but with a check first on the hours in case they start doing morning meetings. We could do something like "if the hour int is above 8 and less than 12 it should be AM, otherwise PM"

Reply:
Yeah, i was not quite sure if 1:30 was PM or AM, since in my country we use the 24 hour format.

Reply:
Sounds good! The 24 hour format makes a lot more sense to be honest, but isn't very common here